sfge/src/main/java/com/salesforce/rules/fls/apex/operations/ValidationConverter.java

```diff
@@ -97,8 +102,8 @@ private ValidationHolder getHolder(ApexValue<?> childApexValue, ProcessFields pr
         return holder;
     }
 
-    private void extractFromList(
-            ProcessFields processFields, ValidationHolder holder, List<ApexValue<?>> apexValues) {
+    private void
+            extractFromList(ProcessFields processFields, ValidationHolder holder, List<ApexValue<?>> apexValues) {
```

Comment:
Did you add this linebreak, or was it added automatically by `spotless`? It doesn't matter, I'm just curious.